_data-prepper/managing-data-prepper/configuring-data-prepper.md

```diff
@@ -100,3 +101,102 @@ usage | Yes | Bytes | Specifies the JVM heap usage at which to trip a circuit br
+| secret_id        | Yes      | String   | The AWS secret name or ARN.                                                                                                                                                                                                                                    |
+| region           | No       | String   | The AWS region of the secret. Defaults to `us-east-1`.                                                                                                                                                                                                         |
+| sts_role_arn     | No       | String   | The AWS Security Token Service (AWS STS) role to assume for requests to AWS Secrets Manager. Defaults to `null`, which will use the [standard SDK behavior for credentials](https://docs.aws.amazon.com/sdk-for-java/latest/developer-guide/credentials.html). |
+| refresh_interval | No       | Duration | The refreshment interval for AWS secrets extension plugin to poll new secret values. See [Secrets refreshment](#secrets-refreshment) for details. Defaults to `PT1H`.                                                                                          |
+
+###### Reference secrets
+
+In `pipelines.yaml`, secret values can be referenced within pipeline plugins using the following formats:
+
+* plaintext: `${{aws_secrets:<YOUR_SECRET_CONFIG_ID>}}`.
+* json (key-value pairs): `${{aws_secrets:<YOUR_SECRET_CONFIG_ID>:<YOUR_KEY>}}`
+
+The secret value reference string format can be interpreted for the following plugin setting data types:
+
+* String
+* Number
+* Long
+* Short
+* Integer
+* Double
+* Float
+* Boolean
+* Character
+
+The following snippet in `pipelines.yaml` uses opensearch sink as an example
+
+```
+sink:
+    - opensearch:
+        hosts: [ "${{aws_secrets:host-secret-config}}" ]
+        username: "${{aws_secrets:credential-secret-config:username}}"
+        password: "${{aws_secrets:credential-secret-config:password}}"
+        index: "test-migration"
+```
+
+Note that the above snippet reference assumes plaintext secret value in `host-secret-config` and json key-value pairs in `credential-secret-config` with both `username` and `password` present as keys. 
+The corresponding AWS secrets extension plugin in `data-prepper-config.yaml` is as follows:
+
+```
+extensions:
+  aws:
+    secrets:
+      host-secret-config:
+        secret_id: <YOUR_SECRET_ID_1>
+        region: <YOUR_REGION_1>
+        sts_role_arn: <YOUR_STS_ROLE_ARN_1>
+        refresh_interval: <YOUR_REFRESH_INTERVAL_1>
+      credential-secret-config:
+        secret_id: <YOUR_SECRET_ID_2>
+        region: <YOUR_REGION_2>
+        sts_role_arn: <YOUR_STS_ROLE_ARN_2>
+        refresh_interval: <YOUR_REFRESH_INTERVAL_2>
+```
+
+###### Secrets refreshment
```

Comment:
Let's not use "refreshment" here. A drink may be a refreshment. :) 

I'd prefer "Refreshing secrets" or "Automatically refreshing secrets".